config/development.json

```diff
@@ -93,5 +93,6 @@
 	"ETHERPAD": {
 		"PAD_URI": "http://localhost:9001/p"
 	},
-	"PROVISIONING_SCHULCONNEX_LIZENZ_INFO_URL": "http://localhost:8888/v1/lizenzinfo"
+	"PROVISIONING_SCHULCONNEX_LIZENZ_INFO_URL": "http://localhost:8888/v1/lizenzinfo",
+	"FEATURE_NEW_LAYOUT_ENABLED": true
```

Comment:
This should only be set when we want to enable this on every developer's machine. Otherwise the devs can enable it in their own `.env` files.

Reply:
I took it out